recipes/discasm/meta.yaml

```diff
@@ -1,38 +1,47 @@
+{% set name = "discasm" %}
 {% set version = "0.1.2" %}
+{% set build = "1" %}
 
 package:
-    name: discasm
-    version: {{ version }}
+  name: {{ name }}
+  version: {{ version }}
 
 source:
   fn: DISCASM-v{{ version }}.tar.gz
   url: https://github.com/DISCASM/DISCASM/releases/download/DISCASM-v{{ version }}/DISCASM-v{{ version }}.tar.gz
   md5: 571b2265018608298752f40f3f375b0f
 
 build:
-    number: 0
-    skip: true  # [osx or py3k]
-    # The reason for skipping osx is that trinity >=2.4 is not available in the osx channel
-    # The python code in DISCASM runs on python 2.7.
+  number: 1
```

Comment:
Use `number: {{ build }}` to be consistent.

Reply:
right. Will do.